### DISCUSSION TITLE
倉頡用重（Z）鍵作萬用字元輸入未知碼

### DISCUSSION THREAD

**Discussion by 307587** [rime/home#1403] [Q&A]
視窗的倉頡有用重（Z）鍵作萬用字元輸入未知碼／模糊碼，如「人一＊口」顯示「倉」等字，如何在小狼毫這樣設？
如果要順便將之加強到「＊日」、「土＊一＊金」、「土＊一月金」等格式（視窗倉頡的模糊碼不可用來查首碼、二四碼、尾二之外的中間碼）又要怎設？

**Comment by Sunpulseqd** [comment]
我也想知道五笔支持模糊码吗？
